pkg/apis/app/v2/kieapp_types.go

```diff
@@ -595,6 +595,7 @@ type TemplateConstants struct {
 // ConsoleTemplate contains all the variables used in the yaml templates
 type ConsoleTemplate struct {
 	OmitImageStream  bool           `json:"omitImageStream"`
+	Omit             bool           `json:"omit"`
```

Comment:
This shouldn't be necessary, since we already have an Omit option for pam components. For reference -
https://github.com/kiegroup/kie-cloud-operator/blob/5b00d9ab040857258aa17b10a679c57e4f09c1c9/pkg/apis/app/v2/kieapp_types.go#L538-L539
https://github.com/kiegroup/kie-cloud-operator/blob/5b00d9ab040857258aa17b10a679c57e4f09c1c9/pkg/controller/kieapp/defaults/defaults.go#L79-L81